index.html

```diff
@@ -3395,6 +3395,32 @@ <h3>Proof Purpose Validation</h3>
 
       </section>
 
+      <section>
+        <h3>Canonicalization Method Security</h3>
+
+        <p>
+The way in which transformation, namely canonicalization, is performed can
+affect the security characteristics of a system. Selecting among these
+canonicalization mechanisms is dependent on the use case being addressed. Often,
+selecting the simplest mechanism that achieves the desired security requirements
+is the best choice. This section attempts to provide simple guidance to help
+implementers pick between the two main canonicalization mechanisms referred to
+in this specification, namely JSON Canonicalization Scheme [[RFC8785]] and
+RDF Dataset Canonicalization [[RDF-CANON]].
+        </p>
+        <p>
+If an application only utilizes JSON and does not depend on any form of RDF
+semantics, then utilizing a cryptography suite that utilizes JSON
+Canonicalization Scheme [[RFC8785]] is most likely the best approach.
```

Comment:
There may be other considerations, so best to just say it's attactive.

```suggestion
Canonicalization Scheme [[RFC8785]] is an attractive approach.
```